README.md

```diff
@@ -1 +1,96 @@
-# Micro-capstone-2
\ No newline at end of file
+# Micro-capstone-2
+
+# Project Name : Microverse Capstone 2
+This a single page food app where you can comment and like your favourite meal. 
+>   
+>
+>
+
+## Built With
+
+- HTML, CSS, Javascript
+- Bundled with webpack
+- Gitflow and Github flow 
+
+
+## Live Demo (Coming soon)
+
+## [Project presentation](https://drive.google.com/file/d/1ZcokS3rfZnWzMZBxaGcTD2gMQ_DWv-oY/view?usp=sharing)
+
+
+## Resources Used
+- Involvement API - https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270
+- Food API - https://foodish-api.herokuapp.com/
+- Resturant Design - https://www.behance.net/gallery/96647171/Foodie-Landing-Page?tracking_source=search_projects%7Cfood%20website%20
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Getting Started
+
+This project is open source and you are free take modify and do with as you wish. To get a local copy up and running follow these simple example steps.
+
+### Prerequisites ♻️
+- Any modern browser.
+- A local version control like git installed on your computer
+- Code editor (Visual studio code, sublime text, notepad++, ...) installed on your computer;
+- Knowledge of  basic of HTML and CSS.
+
+### Setup 🎰
+-  Click the this [link](https://github.com/c3c1l1a/pair-programming-awesome-books.git) to clone down the source code or download it as a [zip file](https://github.com/c3c1l1a/pair-programming-awesome-books/archive/refs/heads/main.zip)
+
+
+### Install 
+- Once you have the code navigate into the root folder and run `yarn install` to get all dependecies required to run Unit tests, linters checks and set up webpack
+
+### Run Unit tests
+- Use the `yarn test` command to run all the unit test
+
+### Run webpack live server
+- use `yarn start` to open the app with webpacks live server
+
+### Build for deployment
+- use `yarn run build` to build files for deployment and distribution
+
+### Run code linters 🧪
+> Run the commands below in the terminal or `git bash`.
+- `yarn hint .` : for the `html` codebase and use of best practices.
+- `yarn stylelint "**/*.{css,scss}"` :  for the `css` codebase and use of best practices. you can add 
+- `yarn eslint .` :  for the `JavaScript` codebase and use of best practices.
+-  > Note that you can add `--fix` keyword to the command for `Css` and/or `JavaScript` to make sure that you avoid all errors comming from your `code editor`
+- To start webpack server run `yarn start`
+
+## Authors
+
+👤 **Cecilia Mukima**
+
+- GitHub: [@c3c1l1a](https://github.com/c3c1l1a/)
+- Twitter: [@cMukima](https://twitter.com/CMukima)
+- LinkedIn: [cecilia-wangui-mukima](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] [Optional] You will need to fix this link before you merge because it does not work at all!